clarity/contracts/pool/fixed-weight-pool.clar

```diff
@@ -1109,3 +1111,19 @@
    )
  )
 )
+
+(define-public (get-weighted-dy (token <ft-trait>) (collateral <ft-trait>) (weight-x uint) (weight-y uint) (dx-to-dy uint) (min-dx (optional uint)) (min-dy (optional uint))) 
```

Comment:
`get-weighted-dy` and `get-balance-x-to-y` are almost identical. Can we please re-write them as

`(define-public (swap (token-x-trait <ft-trait>) (token-y-trait <ft-trait>) (weight-x uint) (weight-y uint) (dx uint) (min-dy (optional uint))`

Reply:
done